source/features/dim-bots.tsx

```diff
@@ -51,10 +57,21 @@ function undimBots(event: DelegateEvent): void {
 }
 
 function init(signal: AbortSignal): void {
-	for (const bot of $$(commitSelectors)) {
+	let isLegacy = false;
+	let commits = $$(newViewCommitSelectors);
+
+	if (commits.length === 0) {
+		isLegacy = true;
+		commits = $$(commitSelectors);
+	}
+
+	for (const bot of commits) {
 		// Exclude co-authored commits
-		if ($$('a', bot.parentElement!).every(link => link.matches(commitSelectors))) {
-			bot.closest('.commit, .Box-row')!.classList.add(dimBots.class);
+		if (!isLegacy || $$('a', bot.parentElement!).every(link => link.matches(commitSelectors))) {
```

Comment:
I don’t think so, the condition excludes coauthored commits with humans, so it likely should not excluded this way.